### DISCUSSION TITLE
How do you add multiple different images into 1 animation?

### DISCUSSION THREAD

**Discussion by lchmoddev** [HaxeFlixel/flixel#2510] [Q&A] [answered]
I am trying to make a game, but I have encountered a problem where I need 2 different .png files to be in one animation, but I can't figure out how to do it.

**Comment by Geokureli** [comment]
I recommend using a spritesheet instead of multiple images, when possible
https://haxeflixel.com/documentation/sprites-and-animation/

**Comment by Cheemsandfriends** [comment, marked as answer]
> I am trying to make a game, but I have encountered a problem where I need 2 different .png files to be in one animation, but I can't figure out how to do it.

I answered this on the Haxe's discord server, but the worst answer that I can give is this:
```haxe
var bitmap = Assets.getBitmapData(Path1);
var bitmap2 = Assets.getBitmapData(Path2);
// max is a function I use for comparing ints, but basically it's `return a < b ? b : a;`
var bitmapDraw = new BitmapData(max(bitmap.width, bitmap2.width), bitmap.height + bitmap2.height, true, 0x00000000);
bitmapDraw.draw(bitmap);
bitmapDraw.draw(bitmap2, new FlxMatrix(1,0,0,1, 0, bitmap.height));

bitmap = bitmapDraw;
```
I don't think it's the best solution but it's one of them.
I'd recommend using a spritesheet as GeoKureli said.
this basically turns the separate images into a spritesheet to use I guess.
